942

0xtr3 opened the pull request:
Close: #942.
Changed global scroll bar colour from green #19aa6e50 to dark brown #402D28.

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

## Summary by CodeRabbit

- **Style**
	- Updated scrollbar handle color to a solid shade of the secondary color
	- Refined visual appearance of scrollbar styling
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Commit:
Close 942.
Changed global scroll bar colour from green #19aa6e50 to dark brown #402D28.

Commit:
Changed global scroll bar colour back to #19aa6e50 so that I can push later.

Commit:
Close 942.
Changed global scroll bar colour from green #19aa6e50 to dark brown #402D28.

ghost commented:
Ready for review.

Marchand-Nicolas reviewed (changes requested):
The opacity is not good (should not be transparent anymore)

Commit:
Merge branch 'lfglabs-dev:testnet' into 942

Commit:
Close 942.
Changed global scroll bar colour from green #19aa6e50 to #402D2850 dark brown to #402D28.

ghost commented:
> The opacity is not good (should not be transparent anymore)

Ready to review. I have changed from `#402D2850` to `#402D28`.

Marchand-Nicolas reviewed (approved):
Very good work, well done! Lgtm
